sandstorm.js

```diff
@@ -22,7 +22,7 @@ const sandstormBoard = {
 
 if (isSandstorm && Meteor.isServer) {
   const fs = require('fs');
-  const Capnp = Npm.require(`capnp`);
+  const Capnp = Npm.require('capnp');
```

Comment:
How this is related to getStartDayOfWeek ?

Reply:
That's another commit and unrelated.

Reply:
Please remove this change.

Reply:
Shall I create a separate pull request for the separate commit "Prettifier fixes"? Thought this should be good enough to keep the changes in two separate commits :) Will change.

Reply:
Do not create any prettifier fixes. Very likely it breaks generating OpenAPI docs.

Reply:
Done.

Reply:
Also, please revert all changes related to new Javascript syntax https://github.com/wekan/wekan/pull/3034 like ?? because if that is used in any Javascript file, not all OpenAPI docs are created https://github.com/wekan/wekan/commit/9ae20a3f51e63c29f536e2f5b3e66a2c7d88c691

Reply:
This time I did not use the operator `??` for exactly this reason.

Reply:
> Do not create any prettifier fixes. Very likely it breaks generating OpenAPI docs.

Shouldn't we add an automatically check for that? (e.g. by using Travis-CI for that)

Reply:
Maybe there could be a check that OpenAPI docs include all parts. Wekan does use Travis-CI but it only checks some syntax.